lux/js/stores/CartStore.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var assign = require( 'object-assign' );
+var lux = require( 'lux.js' );
+var products = require( './ProductStore' );
+
+var CartStore = new lux.Store( {
+    namespace: "cart",
+
+    state: { products: { } },
+
+    handlers: {
+        addToCart: {
+            waitFor: [ 'products' ],
+            handler: function( product ) {
+                var newState = this.getState();
+                newState.products[ product.id ] = (
+                    newState.products[ product.id ] ||
+                    assign( products.getProduct( product.id ), { quantity: 0 } )
+                );
+                newState.products[ product.id ].quantity += 1;
+                this.setState( newState );
+            }
+        },
+        cartCheckout: function() {
+            this.replaceState( { products: {} } );
+        },
+        successCheckout: function( products ) {
+            // this can be used to redirect to success page, etc.
+            console.log( 'YOU BOUGHT:' );
+            if ( typeof console.table === "function" ) {
```

Comment:
@voronianski just to make sure I'm clear - remove the console.table? Or was this referencing the spaces in the arg list and inside the if?